README.md

```diff
@@ -106,6 +106,7 @@ zone.  This role has a trust relationship with the users account.
 | [aws_route53_record.root_MX](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/route53_record) | resource |
 | [aws_route53_record.root_SPF](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/route53_record) | resource |
 | [aws_route53_record.root_acm_rules_CNAME](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/route53_record) | resource |
+| [aws_route53_record.rsaa_dev_CNAME](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/route53_record) | resource |
```

Comment:
Why is this here?  It seems to be outside the scope of this PR.

Reply:
Running the terraform-docs command added it. I'll look into why

Reply:
`terraform-docs` may be looking at the remote state to construct its output. Those resources do exist in the state.  I put them out there so #93 can test their changes too.

Reply:
Looks like this was removed in https://github.com/cisagov/cool-dns-cyber.dhs.gov/pull/92/commits/b56552c9fc3074bb725b54bb94372b601cb00a0f, so I am resolving this conversation.